packages/core/src/lib/lit-core.ts

```diff
@@ -891,7 +891,9 @@ export class LitCore {
       this._epochCache.currentNumber &&
       this._epochCache.startTime &&
       Math.floor(Date.now() / 1000) <
-        this._epochCache.startTime + Math.floor(EPOCH_PROPAGATION_DELAY / 1000)
+        this._epochCache.startTime +
+          Math.floor(EPOCH_PROPAGATION_DELAY / 1000) &&
+      this._epochCache.currentNumber >= 3
```

Comment:
This makes sense, approving as this is assuringly to prevent rolling the epoch back to either of the initial dkgs.

Reply:
exactly :-D